internal/test/builders/container.go

```diff
@@ -61,6 +61,15 @@ func WithPort(privateport, publicport uint16, builders ...func(*types.Port)) fun
 	}
 }
 
+// WithSize adds size in bytes to the container
+func WithSize(size int64) func(*types.Container) {
+	return func(c *types.Container) {
+		if size >= 0 {
```

Comment:
nit; this probably is redundant (perhaps there's cases where we'd would want to _reset_ an earlier set value?)

Reply:
Hmm, that's true 🤔. I originally added this function to update `TestContainerListWithConfigFormat` for better clarity, but I do see your point on redundancy. Would you like to remove it from this PR?

Reply:
🤦‍♂️ ☺️  Oh, actually, erm....
Ignore my comment;  I must've read `if size > 0` 😅☺️

It's fine as it is 👌👍